src/Dexter/dexter_utils.py

```diff
@@ -31,6 +31,7 @@ def process_original_delegators_map(delegator_map, contract_id, snapshot_block,
             delegator_map[delegator] = {}
             delegator_map[delegator]['staking_balance'] = balance
             delegator_map[delegator]['current_balance'] = dexter_liquidity_provider_map[delegator]['current_balance']
+        delegator_map[delegator]['originaladdress'] = contract_id
```

Comment:
Is this by intention not in the "else" case? What is the commented code below. Can it be removed? If the code is a todo please remove and put it in an issue

Reply:
I agree! If one delegator has XTZ directly delegated then the original fee ratio should apply not the one from the Dexter contract, thanks for the review! cleanup done of this file done!